#
# DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
#
# Copyright (c) 2004-2010 Oracle and/or its affiliates. All rights reserved.
#
# The contents of this file are subject to the terms of either the GNU
# General Public License Version 2 only ("GPL") or the Common Development
# and Distribution License("CDDL") (collectively, the "License").  You
# may not use this file except in compliance with the License.  You can
# obtain a copy of the License at
# https://glassfish.dev.java.net/public/CDDL+GPL_1_1.html
# or packager/legal/LICENSE.txt.  See the License for the specific
# language governing permissions and limitations under the License.
#
# When distributing the software, include this License Header Notice in each
# file and include the License file at packager/legal/LICENSE.txt.
#
# GPL Classpath Exception:
# Oracle designates this particular file as subject to the "Classpath"
# exception as provided by Oracle in the GPL Version 2 section of the License
# file that accompanied this code.
#
# Modifications:
# If applicable, add the following below the License Header, with the fields
# enclosed by brackets [] replaced by your own identifying information:
# "Portions Copyright [year] [name of copyright owner]"
#
# Contributor(s):
# If you wish your version of this file to be governed by only the CDDL or
# only the GPL Version 2, indicate your decision by adding "[Contributor]
# elects to include this software in this distribution under the [CDDL or GPL
# Version 2] license."  If you don't indicate a single choice of license, a
# recipient has the option to distribute your version of this file under
# either the CDDL, the GPL Version 2 or to extend the choice of license to
# its licensees as provided above.  However, if you add GPL Version 2 code
# and therefore, elected the GPL Version 2 license, then the option applies
# only if the new code is made subject to such option by the copyright
# holder.
#

enterprise_distributedtx.excep_in_enlist=DTX5001:Exception in enlistComponentResources.
DTX5001.diag.cause.1=Exception occurred while enlisting the resource.
DTX5001.diag.check.1=Make sure that the resource usage is from a standard J2EE component or a life cycle component. If this exception occurs from a standard component, this is an Unexpected Internal Error and please contact Payara Support with the complete error log message.
enterprise_distributedtx.excep_in_delist=DTX5002:Exception in delistComponentResources.
DTX5002.diag.cause.1=Exception occurred while delisting the resource.
DTX5002.diag.check.1=Make sure that the resource usage is from a standard J2EE component or a life cycle component. If this exception occurs from a standard component, this is an Unexpected Internal Error and please contact Payara Support with the complete error log message.
enterprise_distributedtx.rollbackexcep_in_regsynch=DTX5003:RollbackException in registerSynchronization.
DTX5003.diag.cause.1=Trying to register the sync object while transaction is marked for rollback.
DTX5003.diag.check.1=This is an Unexpected Internal Error. Please contact Payara Support with the complete error log message.
enterprise_distributedtx.excep_in_utx_begin=DTX5004:RemoteException in UserTx.begin().
DTX5004.diag.cause.1=Exception occurred while enlisting the component resource.
DTX5004.diag.check.1=Make sure that the resource usage is from a standard J2EE component or a life cycle component. If this exception occurs from a standard component, this is an Unexpected Internal Error and please contact Payara Support with the complete error log message.
enterprise_distributedtx.excep_in_utx_commit=DTX5005:RemoteException in UserTx.commit().
DTX5005.diag.cause.1=Exception occurred while delisting the component resources.
DTX5005.diag.check.1=Make sure that the resource usage is from a standard J2EE component or a life cycle component. If this exception occurs from a standard component, this is an Unexpected Internal Error and please contact Payara Support with the complete error log message.
enterprise_distributedtx.excep_in_utx_rollback=DTX5006:RemoteException in UserTx.rollback().
DTX5006.diag.cause.1=Exception occurred while delisting the component resources.
DTX5006.diag.check.1=Make sure that the resource usage is from a standard J2EE component or a life cycle component. If this exception occurs from a standard component, this is an Unexpected Internal Error and please contact Payara Support with the complete error log message.
enterprise_distributedtx.some_excep=DTX5007:Exception :
enterprise_distributedtx.pooling_excep=DTX5008: Pooling Exception:
DTX5008.diag.cause.1=Exception occurred while closing the connection.
DTX5008.diag.check.1=Check if DB is up and running.
#com.sun.enterprise.transaction.* log messages
transaction.reconfig_txn_timeout_failed=DTX5009:Reconfiguration of transaction timeout failed
DTX5009.diag.cause.1=Invalid integer passed to set the timeout.
DTX5009.diag.check.1=Check the timeout that is being given in the server configuration file.
transaction.monitor.registration_failed=DTX5010:An exception occurred while JTAStats monitoring registration
DTX5010.diag.cause.1=Could not register the JTA statistics for monitoring due to configuration problem.
DTX5010.diag.check.1=This is an Unexpected Internal Error. Please contact Payara Support with the complete error log message.
transaction.monitor.error_while_getting_monitor_attr=DTX5011:Error occurred while getting monitor attribute
DTX5011.diag.cause.1=Requested attribute is not monitorable.
DTX5011.diag.check.1=This is an Unexpected Internal Error. Please contact Payara Support with the complete error log message.
transaction.monitor.error_creating_jtastatsimpl=DTX5012:ClassNotFoundException occurred while creating JTAStats
DTX5012.diag.cause.1=JTAStats implementation class is not found.
DTX5012.diag.check.1=This is an Unexpected Internal Error. Please contact Payara Support with the complete error log message.
transaction.monitor.attribute_not_found=DTX5013:Attribute is not part of transaction monitoring
enterprise_distributedtx.before_completion_excep=DTX5014: Caught exception in beforeCompletion() callback:
DTX5014.diag.cause.1=Exception occurred while calling beforeCompletion() method of the registered synchronization
DTX5014.diag.check.1=Check exception message for the exact cause of the exception
enterprise_distributedtx.after_completion_excep=DTX5015: Caught exception in afterCompletion() callback:
DTX5015.diag.cause.1=Exception occurred while calling afterCompletion() method of the registered synchronization
DTX5015.diag.check.1=Check exception message for the exact cause of the exception
xaresource.recover_error=DTX5016:Error in XA recovery. See logs for more details
DTX5016.diag.cause.1=Exceptions while trying to close connections of handlers.
DTX5016.diag.check.1=Individual resource handlers would have logged exceptions if any, check server.log for more details.
transaction.monitor.tm_null=DTX5017: Cannot construct the probe. Transaction Manager is NULL!
DTX5017.diag.cause.1=Transaction Manager is not set in the StatsProvider.
DTX5017.diag.check.1=This error shouldn't happen. Please file a bug with details about the setup.
enterprise_distributedtx.txbean_null=DTX5018: TBEAN IS NULL FOR: 
DTX5018.diag.cause.1=TransactionAdminBean was not constructed properly
DTX5018.diag.check.1=This error shouldn't happen. Please file a bug with details about the setup.
enterprise_used_delegate_name=DTX5019: Transaction Manager is ready. Using [{0}] as the delegate

